episodes/2-identify-the-problem.Rmd

```diff
@@ -191,26 +177,15 @@ facet_wrap(~species)
 
 ```
 
+Our improved code here looks good. A quick "dim" call confirms we now have all the *Dipodomys* observations, and our plot is looking better. In general, having a 'print' statement or some other output before plots or other major steps can be a good way to check your code is producing intermediate results consistent with your expectations.
```

Comment:
another example of jargon--"a quick dim call" adds cognitive load for learners. I would change that to "Checking the dimensions of our subsetted data frame using the `dim()` function confirms..."